envvarProvider.js

```diff
@@ -1,22 +1,23 @@
-const vscode = require('vscode');
-
-const path = require('path'),
-      fs = require('fs'),
-      { EOL } = require('os');
+const   vscode = require('vscode');
+        path = require('path'),
+        fs = require('fs'),
+        { EOL } = require('os');
 
 const findProjectDir = (fileName) => {
     const dir = path.dirname(fileName);
 
     if (fs.existsSync(dir + '/package.json')) {
         return dir;
     } else {
-        if (dir === '/') {
-            return null;
-        }
-        return findProjectDir(dir);
+        return dir === '/' ? null : findProjectDir(dir);
     }
 }
 
+const isDotenvInDeps = (projectDir) => {
+    const { dependencies, devDependencies } = require(`${projectDir}/package.json`);
+    return dependencies && dependencies.dotenv || devDependencies && devDependencies.dotenv;
```

Comment:
Prevent extension from crashing here thus not processing `process.env` entries if dependencies and devDependencies does not exist (quite uncommon)

Reply:
smart move :ok_hand: